Tools/GNUMake/comps/nvcc.mak

```diff
@@ -134,6 +134,16 @@ ifeq ($(nvcc_forward_unknowns),1)
   NVCC_FLAGS += --forward-unknown-to-host-compiler
 endif
 
+ifeq ($(shell expr $(nvcc_major_version) \>= 11),1)
+ifeq ($(GPU_ERROR_CAPTURE_THIS),TRUE)
```

Comment:
This won't work for the 11.0 RC, but that's fine with me.

Reply:
Yes, I assume nobody is going to use 11.0 RC.